packages/lib/constants.ts

```diff
@@ -21,6 +21,11 @@ export const WEBAPP_URL =
 // So for development purpose, we would stick to localhost only
 export const WEBAPP_URL_FOR_OAUTH = IS_PRODUCTION || IS_DEV ? WEBAPP_URL : "http://localhost:3000";
 
+/**
+ * ⚠️ Server-only! Do NOT import from client-side code.
+ */
+export const RETELL_API_KEY = process.env.RETELL_AI_KEY;
```

Comment:
@Udit-takkar best to remove this, the compiler possibly turns this into a string and compiles it in; constants.ts is definitely included client side.